components/hab/src/command/origin/key/list.rs

```diff
@@ -22,7 +22,9 @@ pub fn start(cache: &Path) -> Result<()> {
         let path = entry.path();
 
         if path.is_file() {
-            println!("{:?}", path.file_name());
+            if let Some(name) = path.file_name() {
+                println!("{:?}", name);
```

Comment:
Maybe you could use this: https://doc.rust-lang.org/stable/std/path/struct.PathBuf.html#method.display